proccon.py

```diff
@@ -0,0 +1,283 @@
+            try:
+                proc_name = proc.ImageFileName.cast(
+                    "string",
+                    max_length=proc.ImageFileName.vol.count,
+                    errors="replace"
+                )
+                
+                proc_pid = proc.UniqueProcessId
+                proc_ppid = proc.InheritedFromUniqueProcessId
+                
+                # Get full path
+                try:
+                    peb = proc.get_peb()
+                    if peb:
+                        process_params = peb.ProcessParameters
+                        if process_params:
+                            image_path = process_params.ImagePathName.get_string()
+                            proc_path = image_path if image_path else proc_name
+                        else:
+                            proc_path = proc_name
+                    else:
+                        proc_path = proc_name
+                except Exception:
+                    proc_path = proc_name
+                
+                # Get create time
+                try:
+                    create_time = proc.get_create_time()
+                except Exception:
+                    create_time = None
+                
+                # Get threads count
+                try:
+                    threads = proc.ActiveThreads
+                except Exception:
+                    threads = 0
+                
+                processes[proc_pid] = {
+                    "name": proc_name,
+                    "pid": proc_pid,
+                    "ppid": proc_ppid,
+                    "path": proc_path,
+                    "create_time": create_time,
+                    "threads": threads,
+                    "children": []
+                }
+                
+            except Exception as e:
+                vollog.debug(f"Error processing process: {e}")
+                continue
+        
+        # Build parent-child relationships
+        for pid, proc_data in processes.items():
+            ppid = proc_data["ppid"]
+            if ppid in processes:
+                processes[ppid]["children"].append(pid)
+        
+        return processes
+    
+    def _generate_dot_content(self, processes: Dict) -> str:
+        """Generate Graphviz DOT format content."""
+        dot_lines = []
+        
+        # Header
+        dot_lines.append("digraph ProcessTree {")
+        dot_lines.append("    rankdir=TB;")
+        dot_lines.append("    node [shape=box, style=filled];")
+        dot_lines.append("    edge [fontsize=10];")
+        dot_lines.append("    ")
+        
+        # Define node styles
+        dot_lines.append("    // Process nodes")
+        
+        # Find root processes (no parent or parent doesn't exist)
+        root_pids = []
+        for pid, proc_data in processes.items():
+            ppid = proc_data["ppid"]
+            if ppid == 0 or ppid not in processes:

```

Comment:
`ppid == 0` can be written in a more pythonic way as `not ppid`, but it's just a stylistic point, not show stopper